assets/wix/Product.wxs

```diff
@@ -292,6 +465,17 @@
                 </RegistryKey>
               </RegistryKey>
             </Component>
+            <Component Id="SaveInstallerProperties" Permanent="yes">
```

Comment:
I don't advise permanent components. This means users will have a hard time changing in the future, and leaves cruft behind in the registration. Perhaps it's better to change the [`MajorUpgrade`](https://wixtoolset.org/docs/v3/xsd/wix/majorupgrade/) element to use `afterInstallExecute`. Not only is it faster to upgrade, it will prefer existing components until the product is removed (not via upgrade).

@TravisEz13 this would need to be well-tested, though. The current behavior removes the old product first then installs the new product. This is inefficient, but safe for refactorings that might otherwise change component GUIDs. Doubt that's happened in a very long time, though - not since I did a bunch of preview vs. stable refactorings a *long* time ago. May not be worth retaining that behavior anymore.